workers/data_refinery_workers/downloaders/requirements.txt

```
@@ -4,12 +4,13 @@
 #
 #    pip-compile --output-file requirements.txt requirements.in
 #
+asgiref==3.2.3            # via django
 certifi==2018.8.24        # via requests
 chardet==3.0.4            # via requests
 coverage==4.5.1           # via coveralls
 coveralls==1.5.1
 django-elasticsearch-dsl==0.5.1
-django==2.2.7
+django==3.0.1
```

Should we use the same version of Django in all projects?

```suggestion
django>=2.2.9
```

---

Good catch. I changed the version number in requirements.in but I didn't change `>=` to `==` so recompiling didn't downgrade it. I've fixed it here and in the processors as well!